content/copilot/index.md

```diff
@@ -0,0 +1,168 @@
+  app is installed in your organization.
+
+### Setup
+
+Select where you want to use the Docker Extension for GitHub Copilot:
+
+{{< tabs >}}
+{{< tab name="Editor or IDE" >}}
+
+For instructions on how to use the Docker Extension for GitHub Copilot in
+your editor, see:
+
+- [Visual Studio Code](https://docs.github.com/en/copilot/github-copilot-chat/copilot-chat-in-ides/using-github-copilot-chat-in-your-ide?tool=vscode)
+- [Visual Studio](https://docs.github.com/en/copilot/github-copilot-chat/copilot-chat-in-ides/using-github-copilot-chat-in-your-ide?tool=visualstudio)
+- [JetBrains IDEs](https://docs.github.com/en/copilot/github-copilot-chat/copilot-chat-in-ides/using-github-copilot-chat-in-your-ide?tool=jetbrains)
+- [Codespaces](https://docs.github.com/en/codespaces/reference/using-github-copilot-in-github-codespaces)
+
+{{< /tab >}}
+{{< tab name="GitHub.com" >}}
+
+No setup is needed for the Docker Extension for GitHub Copilot on GitHub.com.
+Simply go to any repository and start using the chat interface,
+see [Using the extension](#using-the-extension).
+
+{{< /tab >}}
+{{< /tabs >}}
+
+## Using the Extension
+
+The Docker Extension for GitHub Copilot provides a chat interface that you can
+use to interact with the Docker agent. You can ask questions and get help
+Dockerizing your project.
+
+The Docker agent is trained to understand Docker-related questions, and provide
+guidance on Dockerfiles, Docker Compose files, and other Docker assets.
+
+### Editor
+
+1. Open your project in your editor.
+2. Open the Copilot extension or plugin.
+3. Interact with the Docker agent using the `/docker` slash prompt, followed by your question.
+
+![Copilot chat in Visual Studio Code](/copilot/copilot-vscode.gif)
+
+### GitHub.com
+
+To use the Docker Extension in the GitHub web interface:
+
+1. Go to [github.com](https://github.com/) and sign in to your account.
+2. Go to any repository.
+3. Select the Copilot logo in the site menu to open the chat interface.
+
+   ![Copilot chat button](/copilot/copilot-button.png?w=400px)
+
+4. Interact with the Docker agent by tagging `@docker`.
+
+![Docker Copilot extension on GitHub](/copilot/CopilotDemo.gif)
+
+## Example prompts
+
+Here are some examples of the types of questions you can ask the Docker agent:
+
+### Ask general Docker questions
+
+You can ask general question about Docker. For example:
+
+- `@docker what is a Dockerfile`
+- `@docker how do I build a Docker image`
+- `@docker how do I run a Docker container`
+
+### Ask questions about your project
+
+You can ask questions about your project, such as:
+
+- `@docker what is the best way to Dockerize this project`
+- `@docker how do I create a Dockerfile for this project`
+- `@docker how do I create a Docker Compose file for this project`
+
+The Docker agent will outline the steps for you to follow, and where possible,
+automatically raise a pull request with the necessary Docker assets.
+
+### Update Docker assets and code
```

Comment:
Woah, did you try these out (here down)? Did they work?!

Reply:
I did try them - but only some. The agent started having technical difficulties as I was going through these prompts. I will run through them once more.